hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -29,6 +29,14 @@ class BankedContentMetadata(t.TypedDict):
     json: t.NotRequired[dict[t.Any, t.Any]]
 
 
+class BankContentResponse(t.TypedDict):
```

Comment:
ignorable: I think we've found that dataclass also translates, but I'm fine with this